scripts/issue/930/frax_stake_badgerfraxbp.py

```diff
@@ -0,0 +1,42 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+# https://etherscan.io/address/0x5a92EF27f4baA7C766aee6d751f754EBdEBd9fae#code#L722
+MIN_LOCK_TIME = 594000
+
+# https://etherscan.io/tx/0x926d29b92b4b1b2deef70e54e41d4478106eb8062a59ea58c74f628892c05710#eventlog
+PID = 35
+
+vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+
+badger_fraxbp = vault.contract(r.treasury_tokens.badgerFRAXBP_f_lp)
+stkcvx_badger_fraxbp = vault.contract(r.convex.frax.stk_badger_fraxbp)
+
+
+def create_vault():
+    vault.init_convex()
+    vault.convex.create_vault(stkcvx_badger_fraxbp, PID)
+
+    vault.post_safe_tx()
+
+
+def stake_in_vault():
```

Comment:
why the separation into two funcs? can all be done atomically right?

Reply:
vault address resulting from the factory's `CREATE` depends on its internal nonce; hence onfork address might not equal onchain address generated address (another call to the same function might update the nonce between posting and executing).